cloudify_nsx/library/nsx_security_group.py

```diff
@@ -78,6 +79,145 @@ def add_group(client_session, scopeId, name, member, excludeMember,
     return raw_result['objectId']
 
 
+def add_group_exclude_member(client_session, security_group_id, member_id):
+    raw_result = client_session.read(
+        'secGroupObject', uri_parameters={'objectId': security_group_id}
+    )
+
+    common.check_raw_result(raw_result)
+
+    security_group = raw_result['body']
+
+    if 'excludeMember' not in security_group['securitygroup']:
+        security_group['securitygroup']['excludeMember'] = []
+
+    excludeMembers = security_group['securitygroup']['excludeMember']
+    if isinstance(excludeMembers, dict):
+        excludeMembers = [excludeMembers]
+
+    for member in excludeMembers:
+        if member.get("objectId") == member_id:
+            raise cfy_exc.NonRecoverableError(
+                "You already have such member in security group."
```

Comment:
Similar to the comment above, this would probably be better as something like: "Security group {member} already exists in {group_name}"